docs/reference/upgrade/close-ml.asciidoc

```diff
@@ -1,32 +1,33 @@
 [testenv="platinum"]
 
```

Comment:
I don't think the changes in this page are correct (i.e. you'd need to have the third bullet applied to the first paragraph too at the very least).  If the bullet form is desired, can we please separate this page's changes into a new PR and I can pursue them with the appropriate reviewers.

Reply:
I haven't split this out yet, but I will.

Reply:
I've created https://github.com/elastic/elasticsearch/pull/40006